### DISCUSSION TITLE
how to select items to use the batch feature

### DISCUSSION THREAD

**Discussion by MrColumbo** [go-shiori/shiori#817] [Q&A] [answered]
There is a pencil icon which obviously should get used for batch editing. When I click i get an error that nothing is selected but I have no idea how to select bookmarks.

**Comment by Monirzadeh** [comment, marked as answer]
After clicking the pencil icon, you can click the bookmarks you want to select.
Currently you can delete, edit tags and update cache or download ebook in batch mode.

**Reply by MrColumbo** [reply]
thanks  - I always thought I first need to select them ..

**Comment by fmartingr** [comment]
Which browser and operating system are you using? Do you know how to get the browser error console to show up and check if there's any errors in there?
